README.md

```diff
@@ -24,12 +24,13 @@ You need to make your own configuration file for your projects. Example:
     ---
     :external:
       :google_drive:
-        :email: [EMAIL ADDRESS OF YOUR GOOGLE ACCOUNT]
-        :password: [PASSWORD FOR YOUR GOOGLE ACCOUNT]
+        :client_id: [CLIENT ID OF YOUR GOOGLE PROJECT]
```

Comment:
How can we obtain this Client ID and secret? Would you describe the required process in README?